src/backend/config/tools.py

```diff
@@ -12,6 +12,7 @@
     LangChainVectorDBRetriever,
     LangChainWikiRetriever,
     TavilyInternetSearch,
```

Comment:
We recently added a new `backend/community/tools` folder, can you move all logic there instead?